commit 919618d2323473a43865384fecfd5113cc62b0dc
Author: Hans de Goede <hdegoede@redhat.com>
Date:   Wed May 23 15:33:21 2018 +0200

    power: supply: axp288_charger: Fix initial constant_charge_current value
    
    [ Upstream commit f2a42595f0865886a2d40524b0e9d15600848670 ]
    
    We should look at val which contains the value read from the register,
    not ret which is always 0 on a successful read.
    
    Signed-off-by: Hans de Goede <hdegoede@redhat.com>
    Fixes: eac53b3664f59 ("power: supply: axp288_charger: Drop platform_data dependency")
    Signed-off-by: Sebastian Reichel <sebastian.reichel@collabora.co.uk>
    Signed-off-by: Sasha Levin <alexander.levin@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/power/supply/axp288_charger.c b/drivers/power/supply/axp288_charger.c
index 6e1bc14c3304..735658ee1c60 100644
--- a/drivers/power/supply/axp288_charger.c
+++ b/drivers/power/supply/axp288_charger.c
@@ -718,7 +718,7 @@ static int charger_init_hw_regs(struct axp288_chrg_info *info)
 	}
 
 	/* Determine charge current limit */
-	cc = (ret & CHRG_CCCV_CC_MASK) >> CHRG_CCCV_CC_BIT_POS;
+	cc = (val & CHRG_CCCV_CC_MASK) >> CHRG_CCCV_CC_BIT_POS;
 	cc = (cc * CHRG_CCCV_CC_LSB_RES) + CHRG_CCCV_CC_OFFSET;
 	info->cc = cc;
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1101_919618d2323473a43865384fecfd5113cc62b0dc.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1101_919618d2323473a43865384fecfd5113cc62b0dc.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
